app/Adapters/Task.php

```diff
@@ -28,6 +28,42 @@ public function process()
         $this->task->estimatedHours = sprintf('%.2f', $this->task->estimatedHours);
         $this->task->xp = sprintf('%.2f', $this->task->xp);
 
+        $taskStatus = $profilePerformance->perTask($this->task);
+
+        if (!empty($taskStatus)) {
+            $taskEstimatedSeconds = $mappedValues['estimatedHours'] * 60 * 60;
+            $taskDeliveredOnTime = $taskStatus[$this->task->owner]['workSeconds']
+            <= $taskEstimatedSeconds;
```

Comment:
Either add a tab in front of `<= ...` or leave this in the line above - code style issue